### PR TITLE
feat: Add forwarder selection example

### PR DESCRIPTION
**Pull request by denis-tingaikin**
Signed-off-by: Denis Tingaikin <denis.tingajkin@xored.com>

## Motivation

Closes https://github.com/networkservicemesh/deployments-k8s/issues/3940

### CODE REVIEW

File: examples/features/select-fowrarder/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+The example demonstrates how can be declareded forwarder selection in [network service](https://networkservicemesh.io/docs/concepts/architecture/#network-service).
+
+Important points: 
+ - [client](./client.yaml) requests `my-networkservice` service.
+ - [forwarder](./forwarder.yaml) registers itself with label `my_forwarder_capability: true`
+ - [network service](./service.yaml) declaretes network service `my-networkservice` that contains match for label `my_forwarder_capability: true`
+ - [network service-endpoint](./nse-patch.yaml) registers as an [endpoint](https://networkservicemesh.io/docs/concepts/architecture/#network-service-endpoint for service `my-networkservice`
+
+See at example resources in [kustomization.yaml](./kustomization.yaml)
```
Comment (glazychev-art) [inline examples/features/select-fowrarder/README.md:11]: Do we need to add `kustomization.yaml` to this directory?

File: examples/features/README.md [hunk 2]
```diff
@@ -23,3 +23,4 @@ To run any feature example follow steps for [Basic NSM setup](../basic)
 - [Topology aware scale from zero](./scale-from-zero)
 - [NSE composition](./nse-composition)
 - [Exclude prefixes](./exclude-prefixes)
+- [Select forwarder by capability](./select-fowrarder)
```
Comment (glazychev-art) [inline examples/features/README.md:26]: typo: `select-fowrarder` -> `select-forwarder`

File: examples/features/select-fowrarder/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Select forwarder by capability example
+
+The example demonstrates how can be declareded forwarder selection in [network service](https://networkservicemesh.io/docs/concepts/architecture/#network-service).
+
+Important points: 
+ - [client](./client.yaml) requests `my-networkservice` service.
+ - [forwarder](./forwarder.yaml) registers itself with label `my_forwarder_capability: true`
+ - [network service](./service.yaml) declaretes network service `my-networkservice` that contains match for label `my_forwarder_capability: true`
+ - [network service-endpoint](./nse-patch.yaml) registers as an [endpoint](https://networkservicemesh.io/docs/concepts/architecture/#network-service-endpoint for service `my-networkservice`
```
Comment (glazychev-art) [inline examples/features/select-fowrarder/README.md:9]: typo: closing bracket `)`

File: examples/features/select-fowrarder/README.md (hunk 1 above)
Comment (denis-tingaikin) [reply examples/features/select-fowrarder/README.md:11]: yes, note: in this example, we do not have dynamic things. So kustomization.yaml could be static.

File: examples/features/README.md (hunk 2 above)
Comment (denis-tingaikin) [reply examples/features/README.md]: Thanks, fixed.

**Review by glazychev-art — APPROVED** [review]

**Pull request merged**